src/Preflight/Preflight.php

```diff
@@ -321,6 +322,9 @@ public function preflight($argv): array
                 return [$preflightDidRedispatch, $exitStatus];
             }
 
+            // Handle altered root containing a symlink.
+            $alteredRoot = FsUtils::realpath($alteredRoot);
```

Comment:
We tend to be fairly conservative about symlink usage, since they are fraught. However, we have the comparison immediately after this that insists that $alteredRoot must be the same as $root to continue, and $root is the root that we found from our shared `vendor`, so maybe this particular instance of symlink handling is okay. Up to @weitzman